src/SymbolServer.jl

```diff
@@ -27,6 +27,49 @@ mutable struct SymbolServerInstance
     end
 end
 
+function get_general_pkgs()
+    @static if VERSION >= v"1.7-"
+        regs = Pkg.Types.Context().registries
+        i = findfirst(r -> r.name == "General" && r.uuid == UUID("23338594-aafe-5451-b93e-139f81909106"), regs)
+        i === nothing && error("Could not find the General registry")
```

Comment:
Instead of error here and try-catch further down I suggest returning an empty package list from this function. It isn't necessarily something wrong if you don't have General installed.

Reply:
Makes sense. Updated